{"changed": false, "msg": "basic: HTTPSConnectionPool(host='18.232.96.23', port=5986): Max retries exceeded with url: /wsman (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f4481085510>: Failed to establish a new connection: [Errno 111] Connection refused'))", "unreachable": true}